mkdocs/docs/tutorials/load-test-grpc/appserverlessml.md

```diff
@@ -0,0 +1,70 @@
+---
+template: main.html
+---
+
+# K8s App/serverless/ML Frameworks
+
+This tutorial provides examples of using the `load-test-grpc` experiment chart with various Kubernetes app/serverless/ML frameworks. Refer to [`load-test-grpc` usage](usage.md) to learn more about this chart.
+
+!!! tip "Dear Iter8 community" 
+
+    These examples are maintained by members of the Iter8 community, and may become outdated. If you find that something is not working, lend a helping hand and fix it in a PR. More examples are always welcome.
+
+## Knative
+
+???+ note "Before you begin"
+    1. [Install Iter8](../../getting-started/install.md).
+    2. [Install Knative and deploy your first Knative Service](https://knative.dev/docs/getting-started/first-service/). As noted at the end of the Knative tutorial, when you curl the Knative service,
+    ```shell
+    curl http://hello.default.127.0.0.1.sslip.io
+    ```
+    you should see the expected output as follows.
+    ```
+    Hello World!
+    ```
+    3. Update the Knative service deployed above to a gRPC service as follows.
+    ```shell
+    kn service update hello \
+    --image docker.io/grpc/java-example-hostname:latest \
+    --port 50051 \
+    --revision-name=grpc
+    ```
+    4. Download experiment chart.
+    ```shell
+    iter8 hub -e load-test-grpc
+    cd load-test-grpc
+    ```
+
+### 1. Run experiment
```

Comment:
I find this too terse.  This may  be a first entry for a knative user. I think it would be helpful to put some basic text about what the experiment is doing. A link could be included to the usage examples page as well.

Reply:
is this not the example that the current knative doc points to?  I would imagine we want to keep it more clear for any users who to try it.

Reply:
I agree. Will push a change to address this.

Reply:
Done.